### PR TITLE
Fix and enable tests

### PR DESCRIPTION
**Pull request by cnunciato**
Fixes some failing lint checks and enables the `lint` and `test` targets for pull requests.

### CODE REVIEW

File: themes/default/content/blog/pulumiup/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "PulumiUP: the Event for Cloud Engineers"
 date: 2021-04-14
-meta_desc: "PulumiUP is a two hour keynote with cloud engineering industry leaders Adrian Cockcroft, VP of Sustainability Engineering at Amazon, Justin Fitzhugh, VP of Cloud Engineering at Snowflake, Keith Redmond, VP of SaaS Engineering at Fenergo."
```
Comment (cnunciato) [inline themes/default/content/blog/pulumiup/index.md:4]: Was failing the max-length check.

File: themes/default/content/docs/intro/concepts/inputs-outputs.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ let connectionString = pulumi.all([sqlServer.name, database.name])
 {{% /choosable %}}
 {{% choosable language python %}}
 
-In python, you can pass in unnamed arguments to `Output.all` to create an Output list, for example: 
+In python, you can pass in unnamed arguments to `Output.all` to create an Output list, for example:
```
Comment (cnunciato) [inline themes/default/content/docs/intro/concepts/inputs-outputs.md:118]: Was failing the trailing-space check.

File: themes/default/content/resources/getting-started-with-automation-api/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 # Name of the webinar.
-title: "Automation API: Build a Self-Service Infrastructure Platform with Python + Flask"
+title: "Build a Self-Service Infrastructure Platform with Python"
```
Comment (cnunciato) [inline themes/default/content/resources/getting-started-with-automation-api/index.md:3]: Was failing the max length check.

File: themes/default/content/resources/getting-started-with-azure-native/index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 # Name of the webinar.
-title: "Azure: Unlocking Azure with Infrastructure as Software with TypeScript"
+title: "Azure Infrastructure as Software with TypeScript"
```
Comment (cnunciato) [inline themes/default/content/resources/getting-started-with-azure-native/index.md:3]: Was failing the max length check.

**Review by susanev — APPROVED** [review]

**Comment by cnunciato** [comment]
@zchase merging to unblock, but I'll acknowledge any feedback in follow-ups.

**Pull request merged**